pkg/virt-handler/vm_test.go

```diff
@@ -1151,6 +1151,35 @@ var _ = Describe("VirtualMachineInstance", func() {
 			Expect(blockMigrate).To(BeTrue())
 			Expect(err).To(Equal(fmt.Errorf("cannot migrate VMI with non-shared HostDisk")))
 		})
+
+		It("should not allow to migrate with adding default network configuration", func() {
```

Comment:
can  you rename this in a more understandable thing like `should block  migrations if the pod network is connected via  a binding method different than masquerade`?

Apart from that looks good.

Reply:
@rmohr the func `checkNetworkInterfacesForMigration` will not return any error here because the list of interfaces will be empty.
Should we just need to change the test tittle here or return something different from the function?

Reply:
Not sure i can follow. Just ensure that you have a pod nework with the bridge binding set, so that you test that it gets rejected ...

Reply:
@rmohr Changed it